### DISCUSSION TITLE
リリース連絡相談

### DISCUSSION THREAD

**Discussion by Ayato-kosaka** [Ayato-kosaka/react-itinerary#854] [General]
リリース連絡相談チャンネル

**Comment by Ayato-kosaka** [comment]
@Takapy1 
これってどうやってビルド追加できるんだっけ、、


![image](https://github.com/Ayato-kosaka/spelieve/assets/53715839/496b1b7c-125f-42a7-8a79-b1b428e74b0c)

**Reply by Ayato-kosaka** [reply]
@Takapy1 
これ確認お願い〜

**Reply by Takapy1** [reply]
@Ayato-kosaka 
ん、なんかsubmitされてなくないか？

**Reply by Ayato-kosaka** [reply]
@Takapy1 
あ、サブミッションが失敗しているみたい
https://expo.dev/accounts/spelieve/projects/Spelieve/submissions/a5201f87-d635-4efa-af5d-65c1f41d8807

**Reply by Ayato-kosaka** [reply]
@Takapy1 
App Store Connect の契約更新みたいなのの同意が出来てなかったかもしれん！
やり直してみる！

**Reply by Takapy1** [reply]
@Ayato-kosaka 
契約更新してないからsubmit失敗してる感じ？

**Reply by Ayato-kosaka** [reply]
@Takapy1 
多分そうだった！
やり直したらできたから、
アップデート分考えて審査出すわ！

**Comment by Ayato-kosaka** [comment]
ver 2.3

**Reply by Ayato-kosaka** [reply]
@Takapy1 
終わった、、
iOS でアップデートしたらサムネ機能不具合出まくった、、、 :cry: 
早急に障害対応するわ、、、

**Reply by Takapy1** [reply]
@Ayato-kosaka 
なぬ！！！

**Reply by Ayato-kosaka** [reply]
@Takapy1 
```
EAS Build is experiencing a partial outage
Reason: Build service experiencing issues for some users. All information on service status and incidents available at [status.expo.dev](https://status.expo.dev/).
```

EAS のメンテナンスで、 App store connect にビルドをアップロード出来ないんだけど、
野口手動でアップロードする方法わからない？

**Comment by Ayato-kosaka** [comment]
リリース手順

**Reply by Ayato-kosaka** [reply]
@Takapy1 
> (この時点でCICD(eas-build-submit-prod.yml )が起動するが、iOS, Android両方のdevelopment, preview, productionのinternal distributionが動くのでビルド時間がかかるので待機。改善が必要!）

これ、書いてくれてる通りで、ビルド番号のインクリメントとかで、submit 以外のビルドが走っちゃうよね。
今の所対策考えられてないから、ビルドキャンセルで対応して欲しい！

**Comment by Ayato-kosaka** [comment]
v2.3.2

**Reply by Ayato-kosaka** [reply]
@Takapy1 
審査提出ありがとう！
内容確認して問題なかったから、リリースします！

**Reply by Ayato-kosaka** [reply]
@Takapy1 
Android のエラーの件( #870 )解決してなさそうだから確認して欲しい！
（間違えて iOS の方はリリースしちゃった :sweat_drops: 
![image](https://github.com/Ayato-kosaka/spelieve/assets/53715839/8b58347c-6fa2-43cf-95d2-06ed26dd7207)
